init-service/src/main/resources/application-local.yml

```diff
@@ -46,7 +46,7 @@ trial:
       prefix: Mr
       roles:
         - superuser
-      userAccount: some-user-account-id
+      userAccount: 5d995fd0-a1ff-4b2c-ba93-238ba9e349b1
```

Comment:
Is that a secret? Should it go in the config?

Reply:
I don't think it's a secret. It's Mr Bootstrap's AD ID - he only exists for the purpose of proving story 477.

Reply:
So should come from the config-server right?

Reply:
We found another way to approach this so I'm taking it out.